src/core/Akka/Event/LoggingBus.cs

```diff
@@ -226,7 +226,7 @@ public void StartStdoutLogger(Settings config)
         private void SetUpStdoutLogger(Settings config)
         {
             var logLevel = Logging.LogLevelFor(config.StdoutLogLevel);
-            SubscribeLogLevelAndAbove(logLevel, Logging.StandardOutLogger);
+            SubscribeLogLevelAndAbove(logLevel, config.StdoutLogger);
```

Comment:
This line change is what PR title says, right?
Before change in this line (229) `akka.stdout-loglevel` wasn't honored and now, after this line has changed, it is honored, so _off_ means off even during shutdown, right?

Reply:
Now, when you set `stdout-loglevel`, nothing will be outputted by the StandardOutLogger, correct.